src/hooks/useZapExecute.ts

```diff
@@ -174,9 +172,9 @@ const fetchSwapData = async (
   slippageDecimal: number,
 ): Promise<ZapTransactionResponse> => {
   tokenAddress = tokenAddress.toLowerCase();
-  const apiKey = Environment.getZapperApiKey();
+  const sellToken = tokenAddress === "0x0000000000000000000000000000000000000000" ? "ETH" : tokenAddress;
   const response = await fetch(
-    `https://api.zapper.fi/v1/exchange/quote?sellTokenAddress=${tokenAddress}&buyTokenAddress=0x64aa3364f17a4d01c6f1751fd97c2bd3d7e7f1d5&sellAmount=${sellAmount}&slippagePercentage=${slippageDecimal}&network=ethereum&api_key=${apiKey}&ownerAddress=${address}&isZap=true`,
+    `https://api.0x.org/swap/v1/quote?sellToken=${sellToken}&buyToken=0x64aa3364f17a4d01c6f1751fd97c2bd3d7e7f1d5&sellAmount=${sellAmount}&slippagePercentage=${slippageDecimal}&to=${address}`,
```

Comment:
A couple notes for myself:

- [ ] check if we have a rate limit with 0x.
- [ ] also check how buyToken interacts with the olyzap contract.

Reply:
1. [rate limit](https://docs.0x.org/0x-api-orderbook/rate-limiting): 3 requests / second

Reply:
2. it doesn't look like `ownerAddress` queryParams is used https://docs.0x.org/0x-api-swap/api-references/get-swap-v1-quote

Reply:
> 2. it doesn't look like `to` queryParams is used https://docs.0x.org/0x-api-swap/api-references/get-swap-v1-quote

the `to` variable is a response variable. I don't believe adding it as a queryParam has any effect on the response data. It's not a documented accepted request queryParam 
> Also, what do you think about adding `takerAddress`? https://docs.0x.org/developer-resources/api-faqs#how-does-takeraddress-help-with-catching-issues

I tried this yesterday by adding the address of the zap contract as the value, but started getting a `SenderNotAuthorizedError` as the response. In reading through the docs [here](https://docs.0x.org/0x-api-swap/guides/swap-tokens-with-0x-api#specify-a-taker-address-for-your-swaps), it looks like takerAddress only works with non-contract addresses.